eng/scripts/Language-Settings.ps1

```diff
@@ -461,7 +461,7 @@ function GetExistingPackageVersions ($PackageName, $GroupId = $null)
   }
   catch
   {
-    LogError "Failed to retrieve package versions. `n$_"
+    LogError "Failed to retrieve package versions for ${PackageName}. `n$_"
```

Comment:
Do we want this to be an error? Will this trigger for new packages? If so then we probably want to tweak that as we probably only want this to be an error if something other then 404 comes back from the query.